iap/iap_test.py

```diff
@@ -36,6 +38,10 @@
 
 @pytest.mark.flaky
 def test_main(capsys):
+    # It only passes on Kokoro now. Skipping in other places.
+    # The envvar `TRAMPOLINE_CI` will be set once #3860 is merged.
+    if os.environ.get('TRAMPOLINE_CI', 'kokoro') != 'kokoro':
+        pytest.skip('Only passing on Kokoro.')
```

Comment:
What? why?

Reply:
I don't know. Please try locally if you're interested in. I'll file an issue about this.

Reply:
Filed https://github.com/GoogleCloudPlatform/python-docs-samples/issues/3878

Reply:
I'd prefer not adding this skip if the issue is only permissions related

Reply:
Yeah I'll debug this as soon as I get the ownership on the project.